server/actions/users.js

```diff
@@ -168,193 +168,24 @@ export async function getCurrentUser(userId, next) {
 
 // multiple updates, not sure how to handle
 // maybe run in parallel with Promise.all?
-export async function updateUser(
-  email,
-  phone_number,
-  first_name,
-  last_name,
-  date_of_birth,
-  zip_code,
-  address,
-  city,
-  state,
-  notes,
-  userId
-) {
+export async function updateUser(id, user) {
```

Comment:
Careful with this function. The `userId` was actually referring to the adminId of whoever made the change. The createHistoryEventEditProfile keeps an internal record of any admin changes. Sorry, I forgot to mention this, but it looks like you will need to pass an adminId, the userId for the user being updated, and an object that holds the updates.